modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt-rhel-edge_{context}"]
+= Installing the {op-system-rt-kernel} in a {op-system-ostree-first} image
+
+While the real-time kernel is not necessary for low latency workloads, using the real-time kernel can optimize low latency performance and is recommended. You can include it in a {op-system-ostree} image deployment.
+
+.Prerequisites
+* You have a Red Hat subscription enabled on the host that includes {op-system-rt-kernel}.
+* You are using the x86_64 architecture.
+* You configured `osbuild` to use the `kernel-rt` repository.
+//TODO insert link after Eliane's MR is published. Should be right around here: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#appending-a-kernel_creating-system-images-with-composer-command-line-interface
+
+[NOTE]
+====
+The following example blueprint includes references gathered from the previous steps required to configure low latency for a {microshift-short} cluster.
+====
+
+[IMPORTANT]
+====
+A subscription that includes the {op-system-rtk} must be enabled on the host used to build the commit.
+====
+
+.Procedure
+
+. Use the following example blueprint to install the real-time kernel in a {op-system-ostree} image:
++
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "*"
+
+[[packages]]
+name = "microshift-networking"
+version = "*"
+
+[[packages]]
+name = "microshift-selinux"
+version = "*"
```

Comment:
I think we can remove these to reduce the blueprint because they are pull implicitly by the microshift RPM dependency. This should help shorten the file and focus on the new things.

Reply:
Good catch, it means we don't need those in the tests either

Reply:
I see that point, but we also want to show the user what the entire blueprint example looks like. How about shortening this one, but add a Reference module with the whole thing at the end of this docset?

Reply:
Tried it, let's see what it looks like.

Reply:
@ShaunaDiaz , please remove microshift-greenboot, microshift-networking and microshift-selinux `[[packages]]` sections from the blueprint.

The resulting blueprint will remain complete and working. There may be hundreds of packages pulled in implicitly when building blueprint, so we should focus on the current topic and remove the unnecessary ones.